src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -1120,6 +1120,38 @@ public override void VisitConstantPattern(IConstantPatternOperation operation)
             Assert.Same(operation.Value, operation.Children.Single());
         }
 
+        public override void VisitRelationalPattern(IRelationalPatternOperation operation)
```

Comment:
>IRelationalPatternOperation [](start = 52, length = 27)

Consider asserting OperatorKind

Reply:
Will address in [https://github.com/dotnet/roslyn/pull/42548](https://github.com/dotnet/roslyn/pull/42548)

---
In reply to: [394691115](https://github.com/dotnet/roslyn/pull/42474#discussion_r394691115) [](ancestors = 394691115)